File: SDKs/amazonaws.com/polly/2016-06-10/java/lib/src/main/java/openapisdk/models/shared/LexiconDescription.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * LexiconDescription
 * Describes the content of the lexicon.
**/
public class LexiconDescription {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("Attributes")
    public LexiconAttributes attributes;
    public LexiconDescription withAttributes(LexiconAttributes attributes) {
        this.attributes = attributes;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("Name")
    public String name;
    public LexiconDescription withName(String name) {
        this.name = name;
        return this;
    }
}

File: SDKs/amazonaws.com/polly/2016-06-10/python/src/sdk/models/shared/lexicondescription.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import lexiconattributes as shared_lexiconattributes


@dataclass_json
@dataclasses.dataclass
class LexiconDescription:
    r"""LexiconDescription
    Describes the content of the lexicon.
    """
    
    attributes: Optional[shared_lexiconattributes.LexiconAttributes] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Attributes') }})
    name: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Name') }})
    
